cli/src/commands/account.rs

```diff
@@ -328,16 +349,40 @@ fn wait_for_keypress() {
     std::io::stdin().read_exact(&mut single_key).unwrap();
 }
 
+fn check_parent_permissions(path_str: &String) -> Result<()> {
+    #[cfg(target_family = "unix")]
+    {
+        use anyhow::ensure;
+        use std::os::unix::fs::PermissionsExt;
+        use std::path::Path;
+
+        let path = Path::new(path_str);
+        if let Some(parent) = path.parent() {
+            let permissions = parent.metadata()?.permissions().mode();
+            ensure!(permissions & 0o777 == 0o700, "The folder {:?} must be readable only by the owner (0700)", parent);
+        } else {
+            bail!("Parent does not exist for path={}", path_str);
+        }
+    }
+    Ok(())
+}
```

Comment:
isn't this already a part of `commands/start.rs`? in that case it should be moved to `cli/src/lib.rs`, made `pub`lic, and imported in both `start.rs` and here

Reply:
Fixed in https://github.com/AleoNet/snarkOS/pull/3425/commits/9d9eac039c37c1671676045a46e5238eeaf54bee :sunny: